### ISSUE TITLE
 Code fragments cannot be folded or unfolded

### ISSUE THREAD

**Issue by LinkChenzy** [microsoft/vscode#72706]
- VSCode Version:: 1.33.1 (system setup)
- Node.js: 10.2.0
- V8: 6.6.346.32
- OS: Windows_NT x64 10.0.17134

![image](https://user-images.githubusercontent.com/30339294/56498325-bff8c580-6533-11e9-93dd-4aed554bfcff.png)

Steps to Reproduce:

1. plus and minus sign disappear
2.shortcut keys also don't work

**Comment by aeschli** [comment]
Do you have accessibility mode enabled? You can see this on the status line.
Folding is not available in accessibility mode.

**Comment by LinkChenzy** [comment]
Thanks for your reply,but I don't find how to open or close accessibility mode.Do you mean "[Screen reader mode](https://code.visualstudio.com/docs/editor/accessibility#_screen-reader-mode)"?  If two modes are same,I have tried Screen reader mode when all code is expanded,but my screenshot displays 21 lines directly to 103 lines and the middle code is not expanded.
![image](https://user-images.githubusercontent.com/30339294/56641130-35d16e00-66a7-11e9-979f-158bcecdc00a.png)

**Comment by aeschli** [comment]
Yes, Screen Reader Optimized (in the settings it's called Accessibility mode)/

If Screen Reader Optimized is cative you will not see any folding actions (+ or minus).

is the screen shot above with 'Screen Reader Optimized' enabled, or it's the same also without 'Screen Reader Optimized' ?

**Comment by LinkChenzy** [comment]
OK,I get.
But,Sorry,I didn't notice that,just this happens occasionally,after it restarts will be OK.
Next time it heppens, I'll take a screenshot of the whole picture，then back this issue.
Thank you very much!

**Comment by LinkChenzy** [comment]
![TIM截图20190425193131](https://user-images.githubusercontent.com/30339294/56732807-0860ef00-6791-11e9-8b79-e505fad8ae1e.png)
I don't know how this happened.
The same is true of other newly opened files.

**Comment by toferj** [comment]
I came here to report the very same problem. I started noticing it maybe a month or so ago. Every once in a while folding just stops working. Sections that were folded remain so, and I'm unable to unfold them either with the mouse (the controls don't even appear on hover) or with the keyboard shortcuts. The only solution is to exit VSCode and restart it.

I read the bit about accessibility mode. I don't use that at all, so I'm not sure it's relevant (at least I don't _think_ so).

I wish I could provide more information that might be helpful. This happens on both my work and home computers both running Windows 10.

I really just wanted to make sure someone knew about it so that perhaps it could be worked on. 
If there's any information I could grab that would be helpful the next time it happens to me, I'd be happy to provide it if I'm able.

Thanks.
